### PR TITLE
[JetNews]  Bump to alpha03

### PR DESCRIPTION
**Pull request by objcode**
Changes:

- Use new snackbar coroutine-based API
- Work around crash in swipable exposed by re-ordering of `onCommit`
- Migrate to buildSrc `Dependencies.kt` to match other samples.
- Refactor snackbar test to use new testing APIs, and avoid blocking a composable.

### CODE REVIEW

File: JetNews/build.gradle [hunk 1]
```diff
@@ -14,18 +14,18 @@
...
+buildscript {
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.0-alpha08'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath Libs.androidGradlePlugin
```
Comment (JoseAlcerreca) [inline JetNews/build.gradle:27]: JetNews doesn't use buildSrc on purpose. We decided to leave JetNews copypasteable.

**Review by JoseAlcerreca — CHANGES_REQUESTED** [review]
Revert changes to other projects and revert buildSrc changes in JetNews

File: JetNews/.gitignore [hunk 2]
```diff
@@ -1,8 +1,15 @@
 *.iml
 .gradle
 /local.properties
-/.idea
+/.idea/caches
```
Comment (JoseAlcerreca) [inline JetNews/.gitignore:4]: Revert

**Comment by objcode** [comment]
Abandoning this PR and making a new one!

**Pull request closed without merging**